src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -614,6 +618,13 @@ public Instance instance() throws IOException {
     instance.setName(uniqueName());
     instance.setDescription(description);
     instance.setZone(nameFromSelfLink(zone));
+    instance.setMetadata(newMetadata());
+
+    if (!windows) {
+      sshKeyPair = congifureSSHKeyPair(instance, runAsUser);
+    }
+
+    configureStartupScript(instance);
```

Comment:
Can we move that line into `if (!template)` as it was..
If we have template it can have own setup of startup script and we dont want to mess with it..

Reply:
Done.